package.json

```diff
@@ -73,6 +73,7 @@
     "builder-support": "^0.3.0",
     "eslint": "^1.10.1",
     "eslint-config-defaults": "^7.0.1",
-    "eslint-plugin-filenames": "^0.1.2"
+    "eslint-plugin-filenames": "^0.1.2",
+    "json-loader": "^0.5.4"
```

Comment:
`package.json:devDependencies` are for checking the archetype itself, not for _providing_ dev archetype dependencies to upstream projects.

So this should be removed from here and added to `dev/package.json:dependencies` (yes, real deps for the dev published module).